src/main/java/uk/gov/hmcts/reform/notificationservice/data/NotificationStatus.java

```diff
@@ -4,5 +4,6 @@
 
     PENDING,
     FAILED,
-    SENT
+    SENT,
+    MANUALLY_HANDLED,
```

Comment:
worth adding a comment here what this is for ?
`// To be used when there are issues and records needs to be closed manually`